client/src/features/doctor/doctorServices.js

```diff
@@ -0,0 +1,53 @@
+import AuthorizeContract from "../../contracts/Authorize.json";
+
+function doctorServices(params) {
+  this.web3 = params.web3;
+  this.accounts = params.accounts;
+  this.currentUser = params.currentUser;
+  this.address = params.address;
+
```

Comment:
const deployedNetwork = AuthorizeContract.networks[networkId];
    this.instance = new this.web3.eth.Contract(
      AuthorizeContract.abi,
      deployedNetwork && deployedNetwork.address
nếu mà để chung ở trên này thì có được không đỡ phải bị duplicate
    );

Reply:
@sagara11 nó có 1 dòng const networkId = await this.web3.eth.net.getId(); ở trên nên k để chung được